src/Game.java

```diff
@@ -0,0 +1,149 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+
+public class Game {
+    private static final Random RANDOM = new Random();
+
+    private final static int PLAY = 1;
+    private final static int EXIT = 2;
+
+    Scanner scanner = new Scanner(System.in);
+
+    private static final int MAX_ERRORS = 8;
+    private String wordToFind;
+    private char[] wordFound;
```

Comment:
```suggestion
    private char[] foundWord;
```
if you want to use this name. I'd name it `guessedWord`